### ISSUE TITLE
can frida hook the methods that has been confusioned ?

### ISSUE THREAD

**Issue by chensem** [frida/frida#474]
I have a issue about how can frida hook the methods that have been confusioned ..   I tried to hook the methods that not have benn confusioned . it works , but i tried to hook the confusioning methods (the methods's name like Chinese punctuation) . it dosn't work ..  So how can I hook the methods . Thanks .

**Comment by AxDSan** [comment]
What do you mean by Confusioned? you mean obfuscated?

**Comment by chensem** [comment]
yes , it is  obfuscated. I have solved the question .Thanks for your reply.

**Comment by fzts** [comment]
> 是的，它是混淆的。我已经解决了这个问题。谢谢你的回复。

请问是如何解决的，感激不尽。

**Comment by chensem** [comment]
> > 是的，它是混淆的。我已经解决了这个问题。谢谢你的回复。
> 
> 请问是如何解决的，感激不尽。

你这是什么意思

**Comment by fzts** [comment]
> > > 是的，它是混淆的。我已经解决了这个问题。谢谢你的回复。
> > 
> > 
> > 请问是如何解决的，感激不尽。
> 
> 你这是什么意思

想向您请教是如何hook中文函数名的呀，多谢。
![image](https://user-images.githubusercontent.com/28336310/56466505-aaa56d80-6445-11e9-9a58-4642a9c1e8b9.png)
在JEB中能看到中文的函数名，在frida中输出函数名直接乱码。
